### PR TITLE
Appointment request patch

### PR DESCRIPTION
**Pull request by vidya-nv**
Description
OPENSVC-10115 Appointment request patch to booked.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Before : 
image

After: 
image
image
image

### CODE REVIEW

File: lib/resources/r4/appointment_patch.yaml [hunk 1]
```diff
@@ -12,11 +12,33 @@ operations:
...
       <ul>
-        <li>Only "arrived", "checked-in", and "cancelled" are supported</li>
+        <li>Only "arrived", "checked-in", "cancelled", and "booked" are supported</li>
+        <li>This patch operation must be accompanied by the add Slot patch operation when updating status to "booked"</li>
+      </ul>
+  - name: add-slot
+    path: /slot
+    operation: add
+    type: Reference (Slot)
+    url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.slot
```
Comment (MadhurRaj) [inline lib/resources/r4/appointment_patch.yaml:26]: https

Comment (MaxPhilips) [reply lib/resources/r4/appointment_patch.yaml]: https://github.com/cerner/fhir.cerner.com/pull/376/commits/32c74ae4566d27118137c76a3e78d7fd5b8b1ffe

**Comment by ScubaDaniel** [comment]
screencapture-fhir-cerner-millennium-r4-scheduling-appointment-2020-06-16-16_18_02
screencapture-fhir-cerner-millennium-r4-scheduling-appointment-2020-06-16-16_18_02-2

**Comment by DanielWhiteCerner** [comment]
screencapture-fhir-cerner-millennium-r4-scheduling-appointment-2020-06-16-16_18_02
screencapture-fhir-cerner-millennium-r4-scheduling-appointment-2020-06-16-16_18_02-2

**Pull request merged**